.github/workflows/antsibull-docs.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         options:
           - '--use-current --sphinx-theme sphinx_rtd_theme'
           - '--use-current --use-html-blobs --no-breadcrumbs community.crypto community.docker'
-          - '--no-indexes --squash-hierarchy community.crypto --collection-version 2.0.0'
+          - '--no-indexes --squash-hierarchy --intersphinx ansible6:https://docs.ansible.com/ansible/6/ community.crypto --collection-version 2.0.0'
```

Comment:
ansible6 was just an example.  By default it points to `'ansible_devel': ('https://docs.ansible.com/ansible/devel/', (None, '../ansible_devel.inv')`  (the conf_py.j2 file).

Reply:
doh! yeah if I bothered to read the changes a bit better I'd have notice that!  but same second comment applies - given you used core in your testing, do we want to use a core devel here? I don't think we currently generate that one but we could if it's better for this use case?

Reply:
It would be reasonable to describe switching over to the core docs, as being a breaking change (since we currently link to the big-tent docs).  This choice can be overridden by passing `--intersphinx ansible_devel:.......` which I think is reasonable.

Reply:
Right now already the full Ansible `devel` file is included (https://github.com/ansible-community/antsibull-docs/pull/44/files#diff-b032a77af8670b0602c9f139616fb50cbc615a84be5c6b1ecd4b11150b972fdcR36). For this test it doesn't really matter which one is used, if you prefer ansible-core devel I can change it.

Reply:
personally? I prefer Ansible devel, but I may not be the target audience here :-)

Reply:
Since Ansible devel is already included anyway, it doesn't make sense to add it here again.